packages/opds-web-client/src/components/__tests__/DownloadButton-test.tsx

```diff
@@ -43,15 +43,15 @@ describe("DownloadButton", () => {
   it("shows button", () => {
     let button = wrapper.find("button");
     expect(button.props().style).to.deep.equal(style);
-    expect(button.text()).to.equal("Download EPUB");
+    expect(button.text()).to.equal("Download ePub");
```

Comment:
I forgot if we talked about this before, but is it okay to switch to `"ePub"` now?

Reply:
Ah right, I meant to ask this question in my PR - I'm not sure if it okay or not. We can keep EPUB, I just see ePub around the internet most of the time

Reply:
Yes, let's keep `EPUB` for now.

Reply:
Switched it back : )